CPT331.Core/ObjectModel/Offence.cs

```diff
@@ -76,9 +92,16 @@ public string Name
 			{
 				return _name;
 			}
-		}
-
-		public override int GetHashCode()
+        }
+        public int? OffenceCategoryID
```

Comment:
The OffenceCategoryID is Nullable for now, whilst we work on the OffenceCategory data.